plugins/flytekit-kf-tensorflow/README.md

```diff
@@ -8,4 +8,16 @@ To install the plugin, run the following command:
 pip install flytekitplugins-kftensorflow
 ```
 
-_Example coming soon!_
+## Upgrade TensorFlow Plugin
+Tensorflow plugin is now updated from v0 to v1 to enable more configuration options.
+To migrate from v0 to v1, change the following:
+1. Update flytepropeller to v
```

Comment:
v1.6.0. we just cut a release. https://github.com/flyteorg/flyte/blob/master/CHANGELOG/CHANGELOG-v1.6.0.md